src/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ add_library(cppkafka ${CPPKAFKA_LIBRARY_TYPE} ${SOURCES})
 set_target_properties(cppkafka PROPERTIES VERSION ${CPPKAFKA_VERSION}
                                           SOVERSION ${CPPKAFKA_VERSION})
 target_link_libraries(cppkafka ${RDKAFKA_LIBRARY})
+target_include_directories(cppkafka PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../include)
```

Comment:
I think it's better to use `${PROJECT_SOURCE_DIR}` here so you don't have to traverse the path back.

Reply:
heh, I'm a cmake noob. I was just copy-pasting the include path used internally (a few lines above). Your suggestion seems like a good idea

Reply:
Yeah I actually want to go over these CMake files and clean them up soon, I know a little bit more about CMake now than I did when I wrote those.

Reply:
actually, the problem with that is that I'm pulling in cppkafka as a submodule and building it as part of my project tree. ``${PROJECT_SOURCE_DIR}`` will then point to *my* root, not cppkafka's root.

Reply:
I consider this to be a very useful (if not the most useful) use case for linking against other cmake libraries

Reply:
Are you using the project root when you add as a submodule? According to [this](https://cmake.org/cmake/help/v3.0/variable/PROJECT_SOURCE_DIR.html) it should point to the directory where you last called `project` which is in cppkafka's root CMake file. If this doesn't work, no problem. I'll clean this up soon anyway so I'll see if there's an alternative.

Reply:
hm. I may have made some incorrect assumptions. I'll take a closer look

Reply:
I had a typo when testing it. I force-pushed the fix into this PR